charts/dispatch/values.yaml

```diff
@@ -1,63 +1,58 @@
 # Default values for image-manager.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
-global:
-  pullPolicy: IfNotPresent
-  # NOTE: replace with your client id & secret with helm install
-  skipAuth: false
-  #host_ip:
-  #host: dispatch.vmware.com
-  port: 443
-  debug: true
-  image:
-    repository: dispatch-server
-    tag: v0.1.15
-    host: vmware
-  db:
-    backend: postgres
-    host: postgresql
-    port: 5432
-    user: vmware
-    password: dispatch
-    database: dispatch
-    namespace: dispatch
-    release: postgres
-  ingress:
-    enabled: true
-    class: nginx
-    # Used to create Ingress record (should used with service.type: ClusterIP).
-    # host: dispatch.vmware.com
-    # The annotationsPrefix that your ingress controller requires. default - nginx.ingress.kubernetes.io for
-    # nginx ingress controllers.
-    annotationsPrefix: nginx.ingress.kubernetes.io
-    responseHeaders: X-Dispatch-Org
-    annotations:
-      # Specify any additional ingress annotations here. These will be applied to all ingress resources.
-      # kubernetes.io/ingress.class: "nginx"
-  registry:
-    insecure: false
-    uri: docker-docker-registry.docker.svc.cluster.local:5000
-  rabbitmq:
-    username: dispatch
-    password: dispatch
-    port: 5672
-  kafka:
-    brokers:
-    - transport-kafka.dispatch:9092
-  zookeeper:
-    location: zookeeper.zookeeper.svc.cluster.local
-  tls:
-    secretName: dispatch-tls
-  tracer:
-    endpoint:
-  rbac:
-    create: true
-    # Ignored, if rbac.create is true
-    serviceAccountName: default
-  resources: {}
-    # limits:
-    #  cpu: 100m
-    #  memory: 128Mi
-    #requests:
-    #  cpu: 100m
-    #  memory: 128Mi
+# Default values for function-manager.
```

Comment:
Nit: Redundant Comments L4-L6

Reply:
Forgot this one.  Fixed now.  Should be all good to go